assignments/can_drive/can_drive.js

```diff
@@ -1,5 +1,15 @@
 function CanDrive(hasDrivingLiscence, isTired, isSober) {
```

Comment:
We have given only callback assignments you don't have to add a previous assignment in this pull request. So, i didn't looked in this file.